presto-docs/src/main/sphinx/optimizer/logical-planner.rst

```diff
@@ -0,0 +1,35 @@
+*****************
+Logical Planner
+*****************
+
+Presto currently has a rule based optimizer. A given SQL query goes through syntactic
+analysis, semantic analysis, logical plannning, physical planning and then execution.
+Syntax analysis or parsing is used to determine various structures in the given
```

Comment:
Syntactic analysis, or parsing, is used to break down the query into its component structures and check that the query conforms to the syntactic rules of SQL.